pkg/volume/plugins.go

```diff
@@ -272,17 +273,19 @@ type VolumeConfig struct {
 }
 
 // NewSpecFromVolume creates an Spec from an api.Volume
-func NewSpecFromVolume(vs *api.Volume) *Spec {
+func NewSpecFromVolume(vs *api.Volume, namespace string) *Spec {
 	return &Spec{
-		Volume: vs,
+		Volume:    vs,
+		Namespace: namespace,
```

Comment:
I love to get Pod name space in attacher/detacher but I am not sure this is the right place to do it. Same to PV.